ocs_ci/ocs/resources/test_packagemanifest.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf8 -*-
```

Comment:
We should create a `tests` directory to place this file in rather than it living directly in the `resources` module. `ocs_ci/ocs/resources/tests/test_packagemanifest.py`?